### PR TITLE
[ENG-1043] Add error checking to Java getting started gRPC examples

### PR DESCRIPTION
**Pull request by lgoodby**

### CODE REVIEW

File: language-examples/java/grpc/src/main/java/com/vectara/examples/grpc/GrpcBasicOperations.java [hunk 1]
```diff
@@ -298,10 +316,24 @@ public static boolean queryData(
...
+        }
+      }
+      for (var responseSet : response.getResponseSetList()) {
+        for (var status : responseSet.getStatusList()) {
+          if (status.getCode() != StatusCode.OK) {
+            LOGGER.severe("Failure querying corpus: " + status);
+            return false;
+          }
+        }
+      }
```
Comment (amin2718) [inline language-examples/java/grpc/src/main/java/com/vectara/examples/grpc/GrpcBasicOperations.java:332]: Same as above.

**Review by amin2718 — APPROVED** [review]
Looks good. I left a few comments but approved to avoid blocking you.

**Review by tallatshafaat — APPROVED** [review]

File: language-examples/java/grpc/src/main/java/com/vectara/examples/grpc/GrpcApiKeyQueries.java [hunk 2]
```diff
@@ -99,6 +101,20 @@ public static void main(String[] argv) {
...
+      }
+    }
+    for (var responseSet : response.getResponseSetList()) {
+      for (var status : responseSet.getStatusList()) {
+        if (status.getCode() != StatusCode.OK) {
+          LOGGER.severe("Failure querying corpus: " + status);
+          System.exit(1);
+        }
+      }
+    }
```
[the remark this answers is not in the document]
Comment (lgoodby) [reply language-examples/java/grpc/src/main/java/com/vectara/examples/grpc/GrpcApiKeyQueries.java:115]: Done.

File: language-examples/java/grpc/src/main/java/com/vectara/examples/grpc/GrpcBasicOperations.java (hunk 1 above)
Comment (lgoodby) [reply language-examples/java/grpc/src/main/java/com/vectara/examples/grpc/GrpcBasicOperations.java:333]: Done.

**Pull request merged**